hal/src/argon/app_usbd_string_config.h

```diff
@@ -85,7 +85,7 @@
     APP_USBD_STRING_DESC("Argon CDC Mode")
 #else // Argon SoM
 #define APP_USBD_STRINGS_PRODUCT         \
-    APP_USBD_STRING_DESC("ASoM CDC Mode")
+    APP_USBD_STRING_DESC("Argon SoM CDC Mode")
```

Comment:
Why was this necessary? What other measures are needed before we can revert to the new naming?

Reply:
We have a separate story for this, which involves more than just USB strings and I'd like to keep all the changes isolated in a separate PR: [CH32184](https://app.clubhouse.io/particle/story/32184/rename-som-platform-names)

Reply:
It should be "A SoM CDC Mode" eventually.

Reply:
It's already resolved in this PR.

Reply:
Sweet!  Didn't see this was an outdated comment.